### PR TITLE
Added option to override name replacement using Shift when dragging

### PR DESCRIPTION
**Pull request by kenster421**
tokens on the canvas

### CODE REVIEW

**Comment by kenster421** [comment]
I am not knowledgeable enough to translate into other languages. Feel free to pull this in and modify it.

File: scripts/token-mold.js [hunk 1]
```diff
@@ -348,8 +348,9 @@ export default class TokenMold {
   _modifyName(data, actor, sceneId) {
     let name = actor.data.token.name;
 
-    if (this.data.name.replace !== "" && this.data.name.replace !== "nothing")
+    if (["remove", "replace"].includes(this.data.name.replace) && !(this.data.name.baseNameOverride && event.getModifierState("Shift"))) {
```
Comment (kenster421) [inline scripts/token-mold.js:351]: I modified the logic here to be a bit more clear. Name should only be cleared out if the options chosen are "remove" or "replace".
If the setting is enabled in the options, holding shift while dragging a token to the canvas will not remove the actors name. This is useful in the event that you know your group has previously identified a type of creature.

**Comment by kenster421** [comment]
@Moerill bump

**Comment by Moerill** [comment]
@Geekswordsman  is currently maintaining this module.

**Pull request merged**